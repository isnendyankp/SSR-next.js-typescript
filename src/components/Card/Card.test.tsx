/*
    this is a test file for the Card component
*/

import Card from ".";
import { render, screen } from "@testing-library/react";

describe('Card component unit testing', () => {

// this test for card component with explanation should render card with border
    it('should render card with border', () => {
        render(<Card border>Card with border</Card>);
        const cardElement = screen.getByText(/card with border/i);
        expect(cardElement).toMatchSnapshot();
    });
}
);