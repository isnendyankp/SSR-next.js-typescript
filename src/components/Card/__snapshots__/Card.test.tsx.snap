// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component unit testing should render card with border 1`] = `
<div
  class="rounded border-slate-600 border p-8 undefined"
>
  Card with border
</div>
`;
